src/dawn/pipeline_info.h

```diff
@@ -62,12 +66,20 @@ struct RenderPipelineInfo {
 
 struct ComputePipelineInfo {
   ComputePipelineInfo() {}
-  explicit ComputePipelineInfo(::dawn::ShaderModule comp)
-      : compute_shader(comp) {}
+  explicit ComputePipelineInfo(::amber::Pipeline* the_pipeline,
```

Comment:
explicit is only needed if there is only one parameter

Reply:
removed.